airflow/www/app.py

```diff
@@ -200,6 +200,9 @@ def init_views(appbuilder):
             appbuilder.add_link("GitHub",
                                 href='https://github.com/apache/airflow',
                                 category="Docs")
+            appbuilder.add_link("REST API Reference",
+                                href='/api/v1./api/v1_swagger_ui_index',
```

Comment:
```suggestion
                                href='/api/v1/api/v1_swagger_ui_index',
```

Reply:
It's required. We need to refer to an endpoint that is in another blueprint. To do this, we must use the syntax "{blueprint_name}.{endpoint_name}". These two names are generated by connexion, so we have no influence that it do not look cool.

Reply:
aah I thought it was a typo

Reply:
It took me some time to find out these names, but I added a test to detect if something went wrong.